packages/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -120,7 +120,6 @@ export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
             ? operationName
             : commandType;
         const span = plugin._tracer.startSpan(`mongodb.${type}`, {
-          parent: currentSpan,
```

Comment:
Can we remove `const currentSpan = plugin._tracer.getCurrentSpan();` from this file?

Reply:
Tried to make this change but it broke the test. Reverted. Seems like it actually affects the mongo tracing. Maybe in a future PR we should cleanup the mongo plugin to take better advantage of context mechanisms.